### DISCUSSION TITLE
Qodana hangs on indexing

### DISCUSSION THREAD

**Discussion by shyim** [JetBrains/Qodana#70] [Q&A]
See https://github.com/shyim/qodana-report-builder/runs/5087652081?check_suite_focus=true

Project is: https://github.com/shopware/shopware
Qodana Files: https://github.com/shyim/qodana-report-builder/tree/main/default

**Comment by brichbash** [comment]
Could you please add `additional-env-variables: --property=idea.log.config.file=info.xml` to your workflow config? Or if it is possible, please attach logs from `/data/results`

**Reply by shyim** [reply]
I guess this does not help as the job timeouts and the steps after that are not executed 🤔

**Reply by brichbash** [reply]
The property just should make Qodana output more detailed, so that it would be more convenient to investigate

**Reply by shyim** [reply]
That option broke the CI now
https://github.com/shyim/qodana-report-builder/commit/ff596e91c69c8cf16c49849ce72a87202211ed44

https://github.com/shyim/qodana-report-builder/runs/5137636691?check_suite_focus=true

I don't know where this `e` come from

**Reply by brichbash** [reply]
Oh, my apologies. It turned out, the field is responsible for `-e` options of the container, and not for the `--property` that is used in Qodana for JVM properties. Sorry for the inconvenience. We'll add it to the action.

**Reply by brichbash** [reply]
Please, for now add to `qodana.yaml`:
```   
properties:
  idea.log.config.file : info.xml
```
It replaces `--property`

**Reply by shyim** [reply]
are u sure? its not in the list https://github.com/JetBrains/qodana-action/tree/main#configuration

**Reply by brichbash** [reply]
I mean this [qodana.yaml](https://github.com/shyim/qodana-report-builder/blob/main/default/qodana.yaml). The option isn't covered in docs, but it's available in all public images.

**Reply by shyim** [reply]
OHH :D

**Reply by shyim** [reply]
That flag does not work 😅 

You can also clone https://github.com/shyim/platform and then run qodana on it
